### ISSUE TITLE
Font Awesome proper documentation

### ISSUE THREAD

**Issue by incompletude** [nuxt-community/modules#288]
### What problem does this feature solve?

Efficiency when using this package.

### What does the proposed changes look like?

https://github.com/nuxt-community/modules/tree/master/packages/font-awesome

Can the following be added to the documentation:

Are additional packages necessary?
Example usage?
Additional configuration?

Thx

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/modules/issues/c160">#c160</a>)</em></sub></div>

**Comment by martinkrulltott** [comment]
Install according to readme.md, the you use it with eg `<v-icon>fa-facebook</v-icon>`

**Comment by rlam3** [comment]
@pi0 Please update this package. It was really difficult to find out how to use your module.

**Comment by rlam3** [comment]
@incompletude were you able to find a workaround?

**Comment by jcat4** [comment]
I am willing to write and PR the documentation if anyone is capable of figuring out how this module works. The [npmjs](https://www.npmjs.com/package/@nuxtjs/font-awesome) link shows that this package is fairly popular, so I imagine _someone_ out there has figured it out. 🤔

The icons are not available via CSS, traditional and recommended components aren't pulling anything (`v-icon`, `font-awesome-icon`, etc.), and `@fortawesome` isn't installed in node_modules (like the [vanilla package](https://www.npmjs.com/package/@fortawesome/vue-fontawesome) for Vue). I don't know if this is _supposed_ to be a standalone package or if it relies on the existence of the vanilla Vue Font Awesome package or another package or what...

I'm sure this is an awesome module, but making it a bit more accessible would help a lot. I'll dig some more.

**Comment by jcat4** [comment]
Figured it out.

[Link to source](https://github.com/nuxt-community/modules/blob/master/packages/font-awesome/index.js)

This just injects the font awesome source into your app.
Looking at my browser, it looks like it's using Font Awesome 4.7 😢

<img width="694" alt="Screen Shot 2020-01-27 at 2 09 11 PM" src="https://user-images.githubusercontent.com/7866287/73209874-bfb6d780-410e-11ea-8b9f-1fdd0eadbb0d.png">

**TL;DR** Follow directions in the [readme](https://github.com/nuxt-community/modules/blob/master/packages/font-awesome/README.md), and you will have access to Font Awesome 4.7 (not 5) via CSS classes.

Example usage for a [pinterest icon](https://fontawesome.com/v4.7.0/icon/pinterest):
`<i class="fa fa-pinterest"></i>`

[Font Awesome 4.7 docs](https://fontawesome.com/v4.7.0/get-started/)
[Font Awesome 4.7 Icons](https://fontawesome.com/v4.7.0/icons/)

I'll probably whip up some quick changes to the readme and PR later today

**Comment by jcat4** [comment]
Update: It seems this module is more less deprecated. One of the nuxt co-authors said Fontawesome has moved to [this repo](https://www.npmjs.com/package/@nuxtjs/fontawesome).

**Comment by pi0** [comment]
The module is moved here with proper docs and lots of improvements: https://github.com/nuxt-community/fontawesome-module#readme
